docs/DeveloperGuide.md

```diff
@@ -120,10 +126,29 @@ Guarantees: Editing of skill or achievement is successful
   Use case resumes from step 1.
 
 
+### Use case: UC12 - View a user profile item
+
+#### MSS
+
+Precondition: User already has an existing list of user profile items. </br>
+Guarantees: Viewing of user profile item is successful
+
+1.  User requests to view the details of a user profile item.
+2.  InternHunter shows the details of this internship application. </br>
+    Use case ends.
+    
+#### Extensions
+1a. InternHunter detects an invalid input. </br>
+  1a1. InternHunter displays an error message and informs the user of the valid input format. </br>
+  Use case resumes from step 1.
+
 ### Use case: UC13 - Switch tabs
 
 #### MSS
 
+Precondition: User already has an existing list of user profile items. </br>
+Guarantees: Viewing of user profile item is successful
+
```

Comment:
I don't think these are the right precondition and guarantees?

Reply:
yes. These did not have preconditions, I'll remove them.